src/quo2/core.cljs

```diff
@@ -87,6 +87,7 @@
 (def filter quo2.components.selectors.filter.view/view)
 (def skeleton quo2.components.loaders.skeleton/skeleton)
 (def author quo2.components.messages.author.view/author)
+(def display-name quo2.components.messages.author.view/display-name)
```

Comment:
nit: can you move this to `;;;; PROFILE` Section below 🙏  let's keep this file organised :)